lib/winrm-elevated/scripts/elevated_shell.ps1

```diff
@@ -1,8 +1,8 @@
 param([String]$username, [String]$password, [String]$encoded_command, [String]$timeout)
 
 $task_name = "WinRM_Elevated_Shell"
-$out_file = "$env:Temp\winrm_elevated_out.log"
-$err_file = "$env:Temp\winrm_elevated_err.log"
+$out_file = "$env:Temp\winrm_elevated_out-" + "$pid" + ".log"
+$err_file = "$env:Temp\winrm_elevated_err-" + "$pid" + ".log"
```

Comment:
This seems reasonable and has the advantage of tying the log to a specific process. On the hand this might be cleaner and doesn't require pre-execution cleanup. Thoughts?

```
$out_file = [System.IO.Path]::GetTempFileName()
$err_file = [System.IO.Path]::GetTempFileName()
```